src/pe2loaddata/__main__.py

```diff
@@ -69,8 +69,8 @@ def headless(
     # Strip spaces because XML parser is broken
     channels = dict([(str(k).replace(" ", ""), v) for (k, v) in channels.items()])
 
-    if not index_file:
-        index_file = os.path.join(index_directory, "Index.idx.xml")
+    if not index_file:       
+        index_file = os.path.join(index_directory, "Index.xml")
```

Comment:
This should be a glob to catch either Index or Index.idx, please and thank you!

Reply:
Sure, I will change it.  Thanks, Beth!